src/battle/1A5830.c

```diff
@@ -2702,6 +2700,15 @@ API_CALLABLE(RemoveActor) {
 }
 
 API_CALLABLE(DropStarPoints) {
+    /// Star Point multiplier, indexed by actor count.
+    /// +10% multiplier for four actors
+    /// +30% multiplier for three or more actors
+    static s32 multiplier[] = {
```

Comment:
I'd rather this not be static because it hides it from tools and other files. We could call it star point multiplier